### DISCUSSION TITLE
[BUG]debian安装1panel重启后无法自启动

### DISCUSSION THREAD

**Discussion by yhxxm** [1Panel-dev/1Panel#6715] [Q&A]
### 联系方式

yhxxm1@outlook.com

### 1Panel 版本

1.10.18

### 问题描述

服务器重启后 无法自启动面板

### 重现步骤

debian系统 6.6.38内核

**Comment by wanghe-fit2cloud** [comment]
`systemctl status 1panel` 或者 `journalctl -u 1panel -exf` 命令可以查看 1Panel 日志。
